ansible/pbTestScripts/qemuPlaybookCheck.sh

```diff
@@ -81,19 +85,33 @@ usage() {
 defaultVars() {
 	case "$ARCHITECTURE" in
 		"s390x" | "S390X" | "S390x" )
-			echo "s390x selected"; ARCHITECTURE=S390X;;
+			ARCHITECTURE=S390X;;
 		"aarch64" | "arm64" | "ARM64" )
-			echo "aarch64 selected"; ARCHITECTURE=AARCH64;;
+			ARCHITECTURE=AARCH64;;
 		"ppc64le" | "ppc64" | "PPC64LE" | "PPC64" )
-			echo "ppc64le selected"; ARCHITECTURE=PPC64LE;;
+			ARCHITECTURE=PPC64LE;;
 		"arm32" | "ARM32" | "armv7l" | "ARMV7L")
-			echo "arm32 selected"; ARCHITECTURE=ARM32;;
+			ARCHITECTURE=ARM32;;
 		"RISC-V" | "riscv" | "risc-v" | "RISCV" )
-			echo "riscv selected"; ARCHITECTURE=RISCV;;
+			ARCHITECTURE=RISCV;;
 		"" )
 			echo "Please input an architecture to test"; exit 1;;
 		*) echo "Please select a valid architecture"; showArchList; exit 1;;
 	esac
+
+	case "$OS" in
+		"debian8" | "Debian8" | "deb8" )
+			echo "DEBIAN8 selected for $ARCHITECTURE"; OS=DEBIAN8;;
+		"debian10" | "Debian10" | "deb10" )
+			echo "DEBIAN10 selected for $ARCHITECTURE"; OS=DEBIAN10;;
+		"ubuntu18" | "u18" | "Ubuntu18" )
+			echo "UBUNTU18 selected for $ARCHITECTURE"; OS=UBUNTU18;;
+		"debian11" | "deb11" | "Debian11" )
+			echo "DEBIAN11 selected for $ARCHITECTURE"; OS=DEBIAN11;;
+		* )
+			echo "Please use the -o flag to select a supported OS"; showArchList; exit 1;;
```

Comment:
As the `-o` flag is required, could you update the `usage` function, just so it says:
```
	echo "Usage: ./qemu_test_script.sh (<options>) -a <architecture> -o <os>
```
I would've given that as a  'suggested change', but GitHub won't let you do that for pieces of code that haven't been affected  -_-